templates/accounts.go

```diff
@@ -176,16 +176,6 @@ func RemoveAccountKey(address flow.Address, keyIndex int) *flow.Transaction {
 		AddAuthorizer(address)
 }
 
-func bytesToCadenceArray(b []byte) cadence.Array {
```

Comment:
Got a linter error that this wasn't used, so I removed it. Its not connected to the examples passing.